Libraries/CopyLibraries.bat

```diff
@@ -19,7 +19,7 @@ rem Copy SDL libraries.
 xcopy /y %ROOTDIR%\Libraries\SDL\lib\win\x86\SDL2.dll %OUTDIR%
 
 rem Copy Zlib libraries.
-xcopy /y %ROOTDIR%\Libraries\zlib\lib\win\x86\zlib.dll %OUTDIR%
+xcopy /y %ROOTDIR%\Libraries\zlib\lib\win\x86\zlib1.dll %OUTDIR%
```

Comment:
The file is named `zlib1.dll` here:
http://gnuwin32.sourceforge.net/packages/zlib.htm

If I rename it to zlib.dll, the app does not start for me (can't unpack the resources).

Disregard the review thing. That was a misclick.